pkg/webhook/validation/uservalidation.go

```diff
@@ -158,7 +157,7 @@ func checkCRDGroup(group string) bool {
 }
 
 // ValidateMCIdentity returns admission allowed/denied based on the member cluster's identity.
-func ValidateMCIdentity(ctx context.Context, client client.Client, userInfo authenticationv1.UserInfo, resourceNamespacedName types.NamespacedName, resourceKind, mcName string) admission.Response {
+func ValidateMCIdentity(ctx context.Context, client client.Client, userInfo authenticationv1.UserInfo, resourceNamespacedName types.NamespacedName, resourceKind, subResource, mcName string) admission.Response {
 	var mc fleetv1alpha1.MemberCluster
```

Comment:
Hi Arvind! If you don't mind me asking, what's the plan for supporting v1beta1 API?

Reply:
Added TODO